packages/fxa-auth-server/bin/key_server.js

```diff
@@ -135,6 +141,25 @@ async function run(config) {
 
   const profile = require('../lib/profile/client')(log, config, statsd);
   Container.set(ProfileClient, profile);
+
+  if (
+    config.contentful &&
+    config.contentful.apiUrl &&
+    config.contentful.apiKey &&
+    config.contentful.spaceId &&
+    config.contentful.environment
+  ) {
+    const contentfulClient = new ContentfulClient({
+      graphqlApiUri: config.contentful.apiUrl,
+      graphqlApiKey: config.contentful.apiKey,
+      graphqlSpaceId: config.contentful.spaceId,
+      graphqlEnvironment: config.contentful.environment,
```

Comment:
Some of these values were changed as part of a recent change. [Please see here](https://github.com/mozilla/fxa/pull/15862/files#diff-887024629b0aaf52c1338b1c4123e92fba1f360295a3e8bcef3bb7363fafdf86)